### PR TITLE
Added RobCoCertified to masterlist.

### PR DESCRIPTION
**Pull request by ghost**
Thanks Ortham for helping me fix it lol

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -4916,6 +4916,19 @@ plugins:
...
+  - name: 'RobCo Certified.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/43331' ]
+    tag:
+      - Factions
+    dirty:
+      - <<: *quickClean
+        crc: 0x3B28B2FE
+        util: '[FNVEdit v4.0.1i EXPERIMENTAL](https://www.nexusmods.com/newvegas/mods/34703)'
+        itm: 9
+    clean:
```
Comment (Ortham) [inline masterlist.yaml:4930]: It looks like this shouldn't be there.

**Review by Ortham — CHANGES_REQUESTED** [review]
This has less in it than #53, is that intentional?

Also, you don't need to open a new pull request each time you make changes, just make the changes and they'll get added automatically.

File: masterlist.yaml (hunk 1 above)
Comment (ghost) [reply masterlist.yaml:4930]: Yes. I had originally added load order requirements but they are alreaday added as masters, and as such automatically detected. I will however make one update, there should be NVSE requirement listed.

Also, sorry I'm new to this github stuff lol

Comment (Ortham) [reply masterlist.yaml:4930]: A plugin can't be clean and also have wild edits... The wild edit list item should also come under `dirty:`.

**Review by Ortham — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 1 above)
Comment (ghost) [reply masterlist.yaml:4930]: FTFY

Comment (Ortham) [reply masterlist.yaml:4930]: Same as for #57, the fix hasn't appeared here.

Comment (ghost) [reply masterlist.yaml:4930]: Same as #55, appears on my end.

https://github.com/gutfuckllc/falloutnv/commit/021fba321ccbf79238c1fbfba04509423283360e

**Pull request closed without merging**